CONTRIBUTING.md

```diff
@@ -145,3 +145,37 @@ To create a new test:
 
 Snapshots are created and stored in the `integ.example.ts.snapshot` directory and should be committed along with
 the rest of your code.
+
+## Windows Environment Setup
+
+Currently `projen` does not support Windows very well.
+Since this project makes use of `projen`, you will need to setup [Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/about) to contribute as a Windows user.
+
+This section guides you through setting up your Windows machine.
+
+### Installation
+
+Install WSL if you do not have it installed yet:\
+https://learn.microsoft.com/en-us/windows/wsl/install
```

Comment:
It would be good to name the link like `Node Install`.

```suggestion
[WSL Install](https://learn.microsoft.com/en-us/windows/wsl/install)
```

Reply:
see https://github.com/open-constructs/aws-cdk-library/pull/17#discussion_r1579108351